### PR TITLE
Data download page

### PR DESCRIPTION
**Pull request by daveyu1956**
2nd iteration of Data download page following code review
Still to do retrieve area names ref data for dropdown

### CODE REVIEW

File: src/pages/Download/Download.styles.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react';
+import type { ComponentType } from 'react';
+
+
+export const Loading = ({ className, ...props }) => (
+    <p className={ `govuk-body govuk-!-margin-top-6 ${className}` } { ...props }/>
+);
```
Comment (xenatisch) [inline src/pages/Download/Download.styles.js:9]: Check [this](https://github.com/publichealthengland/coronavirus-dashboard/blob/v3-development/src/components/Map/Map.styles.js#L270-L281) out for a more versatile / generic implementation of this.

**Review by xenatisch — CHANGES_REQUESTED** [review]
Just a few minor changes.

File: src/pages/Download/Download.js [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                                            inline={false}
+                                        />
+                                </div>
+                            </div>
+                        </div>          
+
+
+                        <div className="govuk-grid-row govuk-!-margin-top-2">
+                            <div className="govuk-grid-column-one-quarter">
+                                <a  style={isEnabled}
```
[the remark this answers is not in the document]
Comment (daveyu1956) [reply src/pages/Download/Download.js]: done

File: src/pages/Download/Download.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+                                    <h4 className="govuk-heading-s govuk-!-margin-top-1 govuk-!-margin-bottom-0">
+                                        There is a restriction of 10 download requests per approximately 100 seconds.
+                                    </h4>
+                                </div>
+                            </div>
+                        }
+
+
+                        <div className="govuk-grid-row govuk-!-margin-top-2 govuk-!-margin-bottom-2">
+                            <div className="govuk-grid-column-two-thirds">
```
[the remark this answers is not in the document]
Comment (daveyu1956) [reply src/pages/Download/Download.js:167]: done

File: src/pages/Download/Download.js [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+            window.open(href)
+            setThrottleMessage(true);
+            setTimeout(() => { setThrottleMessage(false); }, 20000);
+        };
+
+
+        if ( loading ) return <Loading>Loading&hellip;</Loading>
+
+        return (
+                <div id={ "downloadData" } className={ "govuk-!-margin-top-3" } style={ { display: 'block' } }>
```
[the remark this answers is not in the document]
Comment (daveyu1956) [reply src/pages/Download/Download.js:153]: done

File: src/pages/Download/Download.styles.js (hunk 1 above)
Comment (daveyu1956) [reply src/pages/Download/Download.styles.js]: done

**Review by xenatisch — APPROVED** [review]

**Pull request merged**